sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -72,7 +72,7 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to the execution environment.
-BEAM_CONTAINER_VERSION = 'beam-0.7.0-20170316'
+BEAM_CONTAINER_VERSION = 'beam-0.7.0-20170505'
```

Comment:
We've not created the new workers yet. So there will be one more PR updating them after this one.